public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -353,10 +356,6 @@ export default class FindingDetailsFlyout extends Component<
 
           <EuiSpacer size={'m'} />
 
-          <EuiTitle size={'s'}>
-            <h3>Rule details</h3>
-          </EuiTitle>
-          <EuiSpacer size={'m'} />
```

Comment:
I think we should still keep this, since each accordion would reflect the title of individual rule